docs/Users/FAQ.rst

```diff
@@ -0,0 +1,84 @@
+This is a list of frequently asked questions, aiming to answer any possible
+questions by newcomers or even contributors.
+
+Why did you choose the name?
+============================
+
+coala stands for "COde AnaLysis Application", works well with animals and thus
+is well visualizable, it's easy to memorize.
+
+Is there corporate backing behind coala? What are your intentions?
+==================================================================
+
+coala was founded for fun. coala was, is and will always be free software and is
+developed mostly by students and there's no corporate interest and no CLA. If
+you want to back us, contact us on our `gitter channel <https://gitter.im/coala-analyzer/coala>`__
+.
+
+What sort of analysis does coala do? What languages are supported?
+==================================================================
+
+A list of all analysis routines and supported languages is `here <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst#supported-languages>`__
```

Comment:
I'd break links before the backtick in general, can you change that for this commit?